### ISSUE TITLE
How to make an infinite loop on the last dot ?

### ISSUE THREAD

**Issue by abadikaka** [zbtang/React-Native-ViewPager#47]
Hi guys when i set autplayenable true, when the dots in the last page, and it goes back to first dot seems the view inside of the dots is like reversing back not doing it like infinite loop.
how can i make it like infinite loop ? Both for slide manually and autoplay the slider.. so when goes to last page when i swipe it to right it goes to first dot, same as autoplay enable..

**Comment by nekhaly** [comment]
@abadikaka Did you find a solution for this issue?

**Comment by koenzi132** [comment]
I have the same issue :(

**Comment by abadikaka** [comment]
@nekhaly @koenzi132 no. i need customize myself from this open source, but it was in my first company code. anyway there is another library as well support this one which i forgot the name

**Comment by nekhaly** [comment]
This one was good https://github.com/phil-r/react-native-looped-carousel .. It also allows you to add arrows for moving between items.
